Applications/Classification/jni/main.cpp

```diff
@@ -443,7 +443,7 @@ int main(int argc, char *argv[]) {
     featureVector.resize(feature_size);
     getFeature(img, featureVector);
     nntrainer::Tensor X = nntrainer::Tensor({featureVector});
-    cout << NN.forwarding(X).apply(stepFunction) << endl;
+    cout << NN.forwarding(X, status).apply(stepFunction) << endl;
```

Comment:
Not important. there is `using namespace std` at line 73 though, 
other cout is written like `std::cout` excep this one.

Reply:
that will be deleted later pr. Thanks.